utah-design-system-website/src/react/components/websiteContent/library/components/navigation/verticalMenu/VerticalMenuExampleRender.jsx

```diff
@@ -0,0 +1,61 @@
+import { VerticalMenu } from '@utahdts/utah-design-system';
+import { useImmer } from 'use-immer';
+import { useEffect } from 'react';
+
+/** @typedef {import('utah-design-system-website').VerticalMenuExamplePropsShape} VerticalMenuExamplePropsShape */
+
+/**
+ * @param {object} props
+ * @param {{props: VerticalMenuExamplePropsShape}} props.state
+ * @param {import('react').RefObject<HTMLDivElement>} props.innerRef
+ * @returns {import('react').JSX.Element}
+ */
+export function VerticalMenuExampleRender({
+  state: {
+    props: {
+      childrenMenuType,
+    },
+  },
+  innerRef,
+}) {
+  const [menus, setMenu] = useImmer([
+    // @ts-ignore
+    {
+      header: 'Components',
+      id: 'vertical-menu-sandbox',
+      menuItems: [
+        {
+          id: 'components__text-layout',
+          title: 'Text & Layout',
+          children: [
+            { title: 'Item A',
```

Comment:
!! Discussion topic to see what we like better !!

We could add an eslint rule for "consistent object expression" so that the curly brackets are on separate lines if the object spreads over multiple lines. Sometimes, when the first property of an object is on the same line as the opening bracket, the first item can be lost so that this object appears to be just an object with id and children in it. 

The rule would suggest changing the code to be:
```
{
  title: 'Item A',
  id: 'A',
  children: [
    { title: 'Sub-item A', id: 'G' },
  ],
}
```
Notice the open and close curly brackets are on their own line.

Objects that are all on one line are fine. ie:
```{ title: 'Item B', id: 'B' },```

eslint config change:
```
    "object-curly-newline": [
      "error",
      {
        "ImportDeclaration": {
          "consistent": true
        },
        "ObjectExpression": {
          "consistent": true,
          "multiline": true
        },
        "ObjectPattern": {
          "consistent": true,
          "multiline": true
        }
      }
    ]
```

Reply:
Yes!